podcasts/PodcastListViewController+Tip.swift

```diff
@@ -0,0 +1,78 @@
+import SwiftUI
+import PocketCastsUtils
+
+extension PodcastListViewController: UIPopoverPresentationControllerDelegate {
+    func showRecentlyPlayedSortingTipIfNeeded() {
+        guard
+            Settings.shouldShowRecentlyPlayedSortingTip,
+            FeatureFlag.podcastsSortChanges.enabled,
+            recentlyPlayedSortingTip == nil
+        else {
+            return
+        }
+        if let vc = showRecentlyPlayedSortingTip() {
+            present(vc, animated: true) {
+                Analytics.track(.episodeRecentlyPlayedSortOptionTooltipShown)
+            }
+            recentlyPlayedSortingTip = vc
+        }
+    }
+
+    private func showRecentlyPlayedSortingTip() -> UIViewController? {
+        guard let button = customRightBtn else {
+            return nil
+        }
+        let vc = UIHostingController(rootView: AnyView (EmptyView()) )
+        let idealSize = CGSizeMake(290, 100)
+        let tipView = TipViewStatic(title: L10n.podcastsLibrarySortEpisodeRecentlyPlayedTipTitle,
+                                    message: L10n.podcastsLibrarySortEpisodeRecentlyPlayedTipDescription,
+                                    showClose: true,
+                              onTap: { [weak self] in
+            self?.dismissRecentlyPlayedSortingTip()
+        })
+            .frame(idealWidth: idealSize.width, minHeight: idealSize.height)
+            .setupDefaultEnvironment()
+        vc.rootView = AnyView(tipView)
```

Comment:
Would it make sense to construct the `TipView` first so the UIHostingController `rootView` can just be set once?

```suggestion
        let idealSize = CGSize(290, 100)
        let tipView = TipViewStatic(title: L10n.podcastsLibrarySortEpisodeRecentlyPlayedTipTitle,
                                    message: L10n.podcastsLibrarySortEpisodeRecentlyPlayedTipDescription,
                                    showClose: true,
                              onTap: { [weak self] in
            self?.dismissRecentlyPlayedSortingTip()
        })
            .frame(idealWidth: idealSize.width, minHeight: idealSize.height)
            .setupDefaultEnvironment()

        let vc = UIHostingController(rootView: tipView)
```